net-im/qutim/qutim-0.3.3_p1.ebuild

```diff
@@ -93,6 +93,7 @@ RDEPEND="${CDEPEND}
 DOCS=( AUTHORS INSTALL ChangeLog )
```

Comment:
Surely they are not necessary, but I'd rather keep them around... At least the AUTHORS/ChangeLog.

As for INSTALL, I'm ok to remove it, but frankly speaking, I don't want to make other people to jump through hoops of revbumb/stabilization/rebuilding just because of it. I'd like to postpone it to some a bit more drastic upcoming changes (e.g. dropping of qt-mobility/qtwebkit). Although if you insist, I'll do it now.

After all there is always FEATURES=nodoc for those how don't want tons of useless docs around.